### PR TITLE
Add gethostbyaddr to header

### PR DESCRIPTION
**Pull request by SonicMastr**

### CODE REVIEW

File: newlib/libc/sys/vita/include/netdb.h [hunk 1]
```diff
@@ -112,6 +112,7 @@ struct addrinfo {
 #define NI_WITHSCOPEID      0x00000020
 
 struct hostent *gethostbyname(const char *name);
+struct hostent *gethostbyaddr(const void *__addr, socklen_t __len, int __type);
```
Comment (d3m3vilurr) [inline newlib/libc/sys/vita/include/netdb.h]:
```suggestion
struct hostent *gethostbyaddr(const void *addr, socklen_t len, int ype);
```
could you remove underscores?

Comment (SonicMastr) [reply newlib/libc/sys/vita/include/netdb.h]: I'll do that

**Comment by d3m3vilurr** [comment]
@SonicMastr you have to change variables in the function body too.

**Comment by SonicMastr** [comment]
Sorry. I just saw this now. Will do

**Comment by SonicMastr** [comment]
Should be all good to go now

**Pull request merged**